zips/draft-pm-pure-coinholder-funding.md

```diff
@@ -0,0 +1,253 @@
+    ZIP: XXX
+    Title: Pure Coinholder Funding Model
+    Owners: Tatyana Peacemonger
+            Josh Swihart <josh@electriccoin.co>
+    Credits: Daira-Emma Hopwood
+             Kris Nuttycombe
+    Status: Draft
+    Category: Consensus / Process
+    Created: 2025-02-19
+    License: MIT
+    Pull-Request: <https://github.com/zcash/zips/pull/???>
+
+# Terminology
+
+The key words "MUST", "SHOULD", and "MAY" in this document are to be
+interpreted as described in BCP 14 [^BCP14] when, and only when, they appear in
+all capitals.
+
+The terms "Mainnet" and "Testnet" in this document are to be interpreted as
+defined in the Zcash protocol specification [^protocol-networks].
+
+The terms "Electric Coin Company" (or "ECC"), "Bootstrap Project" (or "BP") and
+"Zcash Foundation" (or "ZF") in this document are to be interpreted as
+described in ZIP 1014 [^zip-1014].
+
+The terms "Zcash Community Grants" (or "ZCG"), "ZCG Committee" and "Deferred
+Dev Fund Lockbox" in this document are to be interpreted as defined in ZIP 1015
+[^zip-1015].
+
+Protocol-defined Ecosystem Funding is funding from sources defined by the Zcash
+Protocol for development of the Zcash ecosystem. At the time of writing,
+Protocol-defined Ecosystem Funding is allocated from a portion of block rewards
+via Funding Streams [^zip-0207] and Lockbox Funding Streams [^zip-2001].
+
+“ZEC” refers to the native currency of Zcash on Mainnet.
+
+# Abstract
+
+This proposal outlines a funding model that gives the coin holders distinct
+voices in determining what, if any, grants are provided to support Zcash’s
+development and community efforts.
+
+Under this model, 20% of the block rewards will accrue to a fund controlled by
+decisions of coin holders, seeded by the Deferred Dev Fund Lockbox. The
+Coinholder-Controlled Fund may be used to distribute grants to ecosystem
+participants, or left at rest.
+
+This system would be activated through until the 3rd halving, allowing enough
+time to determine whether it should be changed or codified for longer.
+
+# Motivation
+
+If no action is taken, in November 2025 funds from block subsidies will stop
+being directed to Zcash Community Grants [^zip-1015-zcg] and to the Deferred
+Dev Fund Lockbox established by ZIP 2001 [^zip-2001]. If the block subsidies
+stop, it will risk a gap in funding of Zcash development organisations, either
+via ZCG grants or via potential future disbursements from the Deferred Dev Fund
```

Comment:
```suggestion
stop, it will risk a gap in funding of Zcash development organisations,
via ZCG grants and via potential future disbursements from the Deferred Dev Fund
```